source/includes/inkling2-reference/_curriculums.html.md

```diff
@@ -113,8 +113,123 @@ graph (input: GameState) {
 
 Another concept that helps with playing Breakout is `KeepPaddleUnderBall`. In the curriculum for this concept, there are two lessons. One of the lessons, `TrackBallAnyPaddle`, uses a wide paddle and is easier to learn. The second lesson, `TrackBallAnyPaddle`, uses a [range expression][5] to vary the paddle width from 1 to 4. The second lesson is more difficult to learn, but by starting with an easier problem, the overall training time is reduced.
 
+### STAR in Inkling
+When using a simulator as the source of training data, each training iteration involves an input to the simulator (an “action”) and an output from the simulator (the new “state”). In addition, a “reward” value is generated along with a “terminal” flag that indicates whether the training episode should be ended. These four piece of data (state, terminal, action, and reward) are sometimes collectively referred to as “STAR”.
+
+#### Reward Functions
+While it’s possible for the simulator code to compute the reward value, it’s often more convenient to implement this reward function in Inkling code. This allows for more rapid experimentation. Simple reward functions can be implemented using the [function][6] capabilities in Inkling.
+
+If a reward function is specified in Inkling, the reward value passed from the simulator is ignored.
```

Comment:
Note, more for the product itself than the docs: it would be good to warn the user that this is happening.